patterns/definitions.owl

```diff
@@ -6067,11 +6284,11 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0100182> "abnormal hindlimb bud"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0100182> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000001> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/XAO_0003066>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0100183> (abnormal archenteron roof)
+# Class: <http://purl.obolibrary.org/obo/XPO_0100183> (abnormal gastrocoel roof plate)
 
```

Comment:
Review the clas 100183 above. Does the name change make sense? @seger

Reply:
yes